client/modules/IDE/components/ErrorModal.stories.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import ErrorModal from './ErrorModal';
+
+export default {
+  title: 'IDE/ErrorModal',
+  component: ErrorModal,
+  argTypes: {
+    type: {
+      options: [
+        'forceAuthentication',
+        'staleSession',
+        'staleProject',
+        'oauthError'
+      ],
```

Comment:
The action wasn't picked up with this 😞 so left that argType.

Reply:
Yeah when I was playing around with it I left the `closeModal` in.  I'm not aware of a trick for those, at least not yet.

FYI, if you want to keep it as a dropdown rather than radio buttons (I don't think it matters?) then you can provide a partial config like `service: {  control: { type: 'select' }  }` which specifies the `control` and it will still infer the `options`.